test-e2e/test/jestGlobalSetup.js

```diff
@@ -0,0 +1,35 @@
+require('babel-register', {
+  presets: ['flow', 'env'],
+  plugins: ['transform-es2015-destructuring', 'transform-object-rest-spread'],
+});
+
+const fs = require('fs-extra');
+
+const {genFixture, packageJson} = require('./helpers');
+
+module.exports = async function jestGlobalSetup(_globalConfig) {
+  global.__TEST_PATH__ = '/tmp/esy-test';
+
+  try {
+    await fs.mkdir(global.__TEST_PATH__);
+  } catch (e) {
+    // doesn't matter if it exists
+  }
+
+  const p = await genFixture(
```

Comment:
Just a fixture that depends on OCaml, if we don't use genFixture here and do all the setup "manually" instead we can get rid of the whole babel hack. That might make sense.